hardhat-tests/test/internal/hardhat-network/helpers/makeCommon.ts

```diff
@@ -0,0 +1,19 @@
+import { Common } from "@nomicfoundation/ethereumjs-common";
+import { rpcQuantityToNumber } from "hardhat/internal/core/jsonrpc/types/base-types";
+import { EthereumProvider } from "hardhat/types";
+
+export async function makeCommon(
```

Comment:
Hardhat removed `Common` from the EDR provider, so we need to mimic creation of `Common` for our purposes. As we're only ever using it for the `chainId` (and `networkId`), this should suffice.

Compared to Hardhat's old implementation, this is ignoring the hardfork. For tests that require on a specifichardfork, we're manually setting the hardfork.

@fvictorio could you give this a sanity check please?